tokens/global/akamai/light.json

```diff
@@ -1,136 +1,938 @@
+        "type": "color"
+      },
+      "90": {
+        "value": "#9C2BAD",
+        "type": "color"
+      },
+      "100": {
+        "value": "#340C3B",
+        "type": "color"
+      },
+      "5": {
+        "value": "#FFF8FF",
+        "type": "color"
+      }
+    },
+    "pink": {
+      "10": {
+        "value": "#FEEEF8",
+        "type": "color"
+      },
+      "20": {
+        "value": "#FCE5F3",
+        "type": "color"
+      },
+      "30": {
+        "value": "#F9D8EC",
+        "type": "color"
+      },
+      "40": {
+        "value": "#F3C6E2",
+        "type": "color"
+      },
+      "50": {
+        "value": "#ECADD4",
+        "type": "color"
+      },
+      "60": {
+        "value": "#E38EC3",
+        "type": "color"
+      },
+      "70": {
+        "value": "#E043A7",
+        "type": "color"
+      },
+      "80": {
+        "value": "#D23197",
+        "type": "color"
+      },
+      "90": {
+        "value": "#CD1D8D",
+        "type": "color"
+      },
+      "100": {
+        "value": "#3B0A2A",
+        "type": "color"
+      },
+      "5": {
+        "value": "#FFF7FC",
+        "type": "color"
+      }
+    },
+    "brown": {
+      "10": {
+        "value": "#F5F2E9",
+        "type": "color"
+      },
+      "20": {
+        "value": "#EEEADD",
+        "type": "color"
+      },
+      "30": {
+        "value": "#E5DFD0",
+        "type": "color"
+      },
+      "40": {
+        "value": "#DAD1BD",
+        "type": "color"
+      },
+      "50": {
+        "value": "#CBBDA4",
+        "type": "color"
+      },
+      "60": {
+        "value": "#B8A383",
+        "type": "color"
+      },
+      "70": {
+        "value": "#978365",
+        "type": "color"
+      },
+      "80": {
+        "value": "#8C795D",
+        "type": "color"
+      },
+      "90": {
+        "value": "#776750",
+        "type": "color"
+      },
+      "100": {
+        "value": "#3B352B",
+        "type": "color"
+      },
+      "5": {
+        "value": "#FBF9F2",
+        "type": "color"
+      }
+    }
+  },
+  "elevation": {
+    "xs": {
+      "value": {
+        "x": "0",
+        "y": "1",
+        "blur": "4",
+        "spread": "0",
+        "color": "rgba(58,59,63,0.2)",
+        "type": "dropShadow"
+      },
+      "type": "boxShadow"
+    },
+    "s": {
+      "value": {
+        "x": "0",
+        "y": "2",
+        "blur": "6",
+        "spread": "0",
+        "color": "rgba(58,59,63,0.2)",
+        "type": "dropShadow"
+      },
+      "type": "boxShadow"
+    },
+    "m": {
+      "value": {
+        "x": "0",
+        "y": "4",
+        "blur": "12",
+        "spread": "0",
+        "color": "rgba(58,59,63,0.2)",
+        "type": "dropShadow"
+      },
+      "type": "boxShadow"
+    },
+    "l": {
+      "value":
+        {
+          "x": "0",
+          "y": "4",
+          "blur": "8",
+          "spread": "0",
+          "color": "rgba(58,59,63,0.08)",
+          "type": "dropShadow"
+        },
```

Comment:
`Elevation.L` was an array (presumably a mistake?)... I'm using the first object but I need confirmation @davyd-akamai